static/temp/foods.json

```diff
@@ -0,0 +1,250 @@
+[
+  {
+    "name": "Banana ripe, slightly ripe, raw",
+    "categoryName": "Fruit and Fruit Juices",
+    "categoryIcon": "EggAltOutlinedIcon",
+    "overview": {
+      "info": "This is a cool food",
+      "NutrientScore": ["A", "B", "C", "D"],
+      "climateImpact": {
+        "title": "Climate Impact",
+        "weight": "20kg co2",
+        "weightTitleSubscript": "Per serving",
```

Comment:
1. Rename `weightTitleSubscript` to `emissionsUnit`
2. Rename value to `per serving` instead of `Per serving`.

Reply:
done